CompilerSource/gcc_interface/gcc_backend.h

```diff
@@ -67,10 +68,4 @@ static inline std::vector<std::string> explode(std::string n) {
   return ret;
 }
 
-inline std::string tolower(std::string x) {
-  for (size_t i = 0; i < x.length(); i++)
-    if (x[i] >= 'A' and x[i] <= 'Z') x[i] -= 'A' - 'a';
-  return x;
-}
```

Comment:
This has been obsolesced by `std::transform` it seems.
`transform(sl.begin(), sl.end(), sl.begin(), ::tolower);`